.travis.yml

```diff
@@ -3,11 +3,11 @@ language: python
 sudo: false
 
 env:
-  - TOXENV=py26
   - TOXENV=py27
-  - TOXENV=py32
-  - TOXENV=py33
   - TOXENV=py34
+  - TOXENV=py35
+  - TOXENV=py36
+  - TOXENV=py37
```

Comment:
Does TravisCI support Python 3.7 out-of-the-box?

Reply:
Sort of. There are workarounds as indicated in [this thread](https://github.com/travis-ci/travis-ci/issues/9815). [Pyramid is in a holding pattern](https://github.com/Pylons/pyramid/issues/3312#issuecomment-403138089).

Reply:
I adapted the `travis.yml` from Webob.